reddit-background

```diff
@@ -52,6 +52,10 @@ RE_RESOLUTION_DISPLAYS = re.compile("Resolution: (\d+)\sx\s(\d+)")
 _VERBOSITY = 0
 _DOWNLOAD_DIRECTORY = None
 _IMAGE_COUNT = 0
+_LINUX = False
+if _LINUX:
+    from Xlib import X, display
```

Comment:
These imports could be moved to the top import section and wrapped in a try/except to automatically determine whether we're a linux system. Also, since `X` and `display` are common-ish names, it'd probably be better to use namespaces to access them rather than make them globals, i.e. access like `Xlib.X` rather than just `X`. Something like:

    try:
        import Xlib
        import Xlib.ext
    except ImportError:
        _LINUX = False
    else:
        _LINUX = True